documentation/0030-node-process-nexttick/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Understanding process.nextTick()'
+description: 'The Node.js process.nextTick function interacts with the event loop in a special way'
+author: flaviocopes
+---
+
+As you try to understand the Node.js event loop, one important part of it is `process.nextTick()`.
+
+Every time the event loop takes a full trip, we call it a tick.
+
+When we pass a function to `process.nextTick()`, we instruct the engine to invoke this function at the end of the current operation, before the next event loop tick starts:
+
+```js
+process.nextTick(() => {
+  //do something
+})
+```
+
+The event loop is busy processing the current function code.
+
+When this operation ends, the JS engine runs all the functions passed to `nextTick` calls during that operation.
+
+It's the way we can tell the JS engine to process a function asynchronously (after the current function), but as soon as possible, not queue it.
+
+Calling `setTimeout(() => {}, 0)` will execute the function in the next tick, much later than when using `nextTick()`.
```

Comment:
`setTimeout(() => {}, 0) will execute the function in the next tick, much later than when using nextTick()` is a bit confusing when we use just the word `next tick` to compare the two. We can go with `setTimeout(() => {}, 0) will execute the function at the end of next tick, much later than when using nextTick() which prioritizes the call and executes it just before the beginning of the next tick`

Reply:
Good catch!